test/e2e/workflow/deployment_test.go

```diff
@@ -30,8 +30,13 @@ var _ = Describe("NetworkAddonsConfig", func() {
 			func(configSpec cnao.NetworkAddonsConfigSpec, components []Component) {
 				testConfigCreate(gvk, configSpec, components)
 
+				timeout := CheckImmediately
+				if len(components) > 1 {
+					By("allowing eventual consistency on co-dependant components deployments")
+					timeout = 10 * time.Second
+				}
```

Comment:
what about just always allow 10s ?

Reply:
yeah I tend to agree..

Reply:
nit: maybe the commit message of that need to be changed please